docs/pages/api-reference/graphql-query.mdx

```diff
@@ -0,0 +1,13 @@
+---
+description: "API reference for the GraphQL query"
+---
+
+# GraphQL query API
```

Comment:
I think for now it makes sense to move this under "Guides" and call it "Query the API". Then I'll expand that page to explain how to sort, paginate, and filter (the filter argument in particular needs documentation).